rasa/core/nlg/response.py

```diff
@@ -75,12 +75,20 @@ def _responses_for_utter_action(
         )
 
         if conditional_channel:
+            logger.debug(
+                "Selecting response variation with condition: "
```

Comment:
Did you intentionally choose to keep `response variation` in singular? Because actually, multiple eligible variations could be selected here then randomly picked in a different method.

Reply:
Ah no, not intentional. I'll get that updated 👍